src/Microsoft.ML.Dnn/ImageClassificationTrainer.cs

```diff
@@ -390,10 +390,10 @@ public sealed class Options : TrainerInputBaseWithLabel
             public Action<ImageClassificationMetrics> MetricsCallback = null;
 
             /// <summary>
-            /// Indicates the path where the newly retrained model should be saved.
+            /// Indicates the path where the models get downloaded and cache files saved
```

Comment:
Should probably explain the default behavior, so callers know what to expect if this isn't specified.